PyInstaller/utils/win32/winmanifest.py

```diff
@@ -921,9 +927,11 @@ def toxml(self, encoding="UTF-8"):
         # version-encoding-standalone (standalone being optional), otherwise 
         # if it is embedded in an exe the exe will fail to launch! 
         # ('application configuration incorrect')
-        xmlstr = domtree.toxml(encoding).replace(
-            '<?xml version="1.0" encoding="%s"?>' % encoding, 
-            '<?xml version="1.0" encoding="%s" standalone="yes"?>' % encoding)
+        xmlstr = domtree.toxml(encoding)
+        xmlstr = xmlstr.decode().strip(os.linesep).replace(
```

Comment:
Why is it necessary to decode the string here? I should be UTF-8 anyway which is compatible to this replace. And below the encode is missing.
Why you need to strip os.linesep ?